Fix missing directory error

yhoiseth opened the pull request:
Fixes #1779.

yhoiseth committed:
Fix missing directory error

tesuji commented on README.md:
```diff
@@ -73,6 +73,7 @@ $ rustup completions bash > ~/.local/share/bash_completion/completions/rustup
 $ rustup completions bash > $(brew --prefix)/etc/bash_completion.d/rustup.bash-completion
 
 # Fish
+$ mkdir ~/.config/fish/completions
```
We might need `-p` flag then.

yhoiseth replied:
Yeah, you're right. I'll add it.

yhoiseth committed:
Create parent directories if necessary

kinnison committed:
help: Update completions help to include mkdirs

In order to ensure that people have the completions directory
in existence before trying to write to it, include `mkdir -p`
commands in the completions help.

Signed-off-by: Daniel Silverstone <dsilvers@digital-scurf.org>
